tech-specs/src/explore/design/intent_gossip/intent_gossip.md

```diff
@@ -0,0 +1,51 @@
+# Intent gossip network
+
+The intent gossip network enables counterparty discovery for bartering. The
+users can express any sort of intents that might be matched and transformed into
+a fulfillable transaction on the Anoma ledger.
+
+An [intent](./intent.md) describes the desire of a user, from asset exchange to a
+green tax percent for selling shoes. These intents are picked up by a matchmaker
+that composes them into transactions to send to the ledger network. A matchmaker
+is optionally included in the intent gossip node.
+
+Each node connects to a specified intent gossip network, either a public or a
+private one. Anyone can create their own network where they decide all aspects
+of it: which type of intents is propagated, which nodes can participate, the
+matchmaker logic, etc. It is possible, for example, to run the intent gossip system
+over bluetooth to have it off-line.
+
+An intent gossip node is a peer in the intent gossip network that has the role
+of propagating intents to all other connected nodes.
+
+The network uses the
+[gossipsub](https://github.com/libp2p/specs/tree/master/pubsub/gossipsub)
```

Comment:
let's use canonical URLs to avoid 404 if this ever moves. (you can press `y` on a github page to get it). 
```suggestion
[gossipsub](https://github.com/libp2p/specs/tree/512accdd81e35480911499cea14e7d7ea019f71b/pubsub/gossipsub)
```